Starting up client...
BaseClient hostname: ec2-54-186-71-144.us-west-2.compute.amazonaws.com port: 8989
verify of signature for wallet_id failed
Client established a remote connection over SSL
sending payload: {"command":"register_wallet","args":{"name":"test","key":"-----BEGIN RSA PUBLIC KEY-----\nMIGJAoGBAMxiVT5fE68rsP0t6xYSBoqRt3xK1hNJv0cB2O82v9ZMkDc29ZIVjKii\nbaoe6edG5HbPGITqGjJxImn6StK7lbz1KpgKGZ5h9q40btPTuySoxaSIqcXqEAzN\nvUlzby/nni5JRciP3RGqxpapExHqOpbkISV4sy7zEa4MW6rxZzXBAgMBAAE=\n-----END RSA PUBLIC KEY-----\n","signature":"282a8f51eca47886cc451a1e2f5490baa69ee27fb7b49037527058d6fad2a77dd4b97f84a7183f8129e9861b5edf369a87470d8e9fa03ddcf81db4944d49712feb9d631eaf9f1f6aeb52676ed725362a4e431374608613d202d358561cadb54ffccf2b5e8bc88f30c580df0147fa029e474820eea4dc696e74c44e99498da3ec"}}
incoming from server: {"wallet_id": "dc589641c33b66ef62f7d1389d1c09e92f7563989f68061567ea6cffe5230204"}
getting current challenge
sent challenge request
incoming from server: {"challenge_id": 466, "hash_prefix": "48ae", "last_solution_hash": "a41d6f5595271d24f70a3a35e7d7e0990126c3cfd4af223c93d6c3a517185c9c", "parameters": {"nb_elements": 100}, "challenge_name": "reverse_sorted_list", "time_left": 108}
solving challenge
